brain/task_executor.py

```
@@ -496,6 +737,147 @@ async def _execute_mcp(
+                        task_id=task_id,
+                        has_task=True,
+                        task_description=task_description,
+                        execution_method='user_plugin',
+                        success=False,
+                        error=f"Trigger endpoint returned status {r.status_code}",
+                        result={"status_code": r.status_code, "text": text},
+                        tool_name=plugin_id,
+                        tool_args=plugin_args,
+                        reason=getattr(up_decision, "reason", "") or "trigger_failed"
+                    )
+        except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)